src/main/java/net/sf/jabref/gui/util/FallbackExceptionHandler.java

```diff
@@ -0,0 +1,24 @@
+package net.sf.jabref.gui.util;
```

Comment:
This does not belong in the gui package, but in JabRef's root package, where the classes that use it reside. Please move it.